data/mods/Russ_Extra_Stuff/Readme.txt

```diff
@@ -0,0 +1,49 @@
+Rusty's stuff version 0.9
+by Russell A.
+mysticinmaine@gmail.com
+
+
+all items either have recipes in the pre-existing books, or are generated on the map, or both.
+
+tools:
+small water purifier
+advanced flashlight
+
+--vehicle parts:
+small kitchen unit (foldable)
+superalloy folding frame
+micro-fridge (foldable)
+small controls (foldable)
+small muffler (foldable)
+dozer bucket (push over some trees why dont you?)
+blue and green cloth wall (foldable)
+micro-reactor
+small and large atomic storage batteries + sturdy versions
+medium electric engine - fills a gap in the vanilla game
+atomic foot pedals - zoom!
+atomic motor - only needs a trace battery charge to start up.
+atomic headlight - reinforced headlight takes only trace battery charge to run.
+tiny advanced solar panel - a solar panel for your four wheeler or electric bicycle.
+
+--guns n ammo
+RWS 34 Diana - air powered 22 pellet rifle.
+Crossman 1322 (.22 air pistol)
+airgun conversion kit 22->177
+.22 and 177 pellets - for the airgun.
+--containers
+Bucket (activate to switch between liquid storage mode or hauling items while wearing on your person)
+--armor:
+advanced power armor hauling frame - more storage than vanilla
+shield generator (can be worn with power armor) - reminiscent of out of this world.  flavor contributions appreciated.
+(the shield generator can be turned on, then activated again for a heat-generating anti-radiation function.
+
+--vehicles pregenerated:
+survivor's bike - someone cobbled this together with advanced parts.
+
+Notes:
+-atomic footpedal atomic headlight and motor: 
+These items are great for small bicycles that need a headlight or motor but don't have a way to generate electricity (e.g. to be economical with weight).  These items are all foldable. these require 10 plutonium cells so they are mid-game-ish.  theyre great for carrying with you as you run around in a city, you can attach the atomic motor to a vehicle and drive away if there is a trace battery charge left in the subject vehicle.
+
+anyway so these items are not perfect and may need rebalancing.
+
+this is an experimental mod.  Use/modify/redistribute to your satisfaction, just mention me in the credits.  Use github.
```

Comment:
The license we use is Creative Commons (to be specific, CC-BY-SA 3.0), we're not accepting contributions under other licenses, so if you consider this line a requirement and not a request, we won't be merging it.